### PR TITLE
Phoenix

### PR DESCRIPTION
**Pull request by dktapps**
_This is only the beginning._

**Current compatible MCPE versions**: 0.15.9 alpha.

This pull request includes various changes cherry-picked from a number of open PRs (#4199, #4178).

Full lists of changes can/will be able to be seen on Phoenix's [releases](https://github.com/iTXTech/Phoenix/releases).

N.B. I am aware that this pull request breaks most of the contribution guidelines. However, I hope that should anyone return to this repository and review this, they may overlook the fact that there are/will be hundred/thousands of changes in this PR from Phoenix. Phoenix was created to help PocketMine through the dark ages.

_N.B.2. There are various changes that have been made to Phoenix to distinguish it from PocketMine to avoid confusion, such as https://github.com/PocketMine/PocketMine-MP/commit/45024357b6fcdff009a527b94e56432771186251, https://github.com/PocketMine/PocketMine-MP/commit/77509eb4db100f335ac84bd86b83f9fc0bc31fe6, https://github.com/PocketMine/PocketMine-MP/commit/ce376a105e586ee195349aea083553e7a5c383e7, https://github.com/PocketMine/PocketMine-MP/commit/4565c3cfb4e30cbc8af2232f54d63fddf54e1115. If anyone comes back and reviews this, those changes will need to be removed before merge._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,31 @@
-# ![PocketMine-MP](http://cdn.pocketmine.net/img/PocketMine-MP-h.png)
-[![Build Status](https://travis-ci.org/PocketMine/PocketMine-MP.svg?branch=master)](https://travis-ci.org/PocketMine/PocketMine-MP)
+# *Phoenix*
+[![Build Status](https://travis-ci.org/iTXTech/Phoenix.svg?branch=master)](https://travis-ci.org/iTXTech/Phoenix)<br>
+__Risen from the ashes of PocketMine-MP__
 
+## About
+Phoenix is a server software for Minecraft Pocket Edition forked from PocketMine-MP 1.6.
```
Comment (iPocket) [inline README.md]: Wow, there seems to be nothing wrong here.

Comment (dktapps) [inline README.md]: Detailed in the description.

**Comment by archie426** [comment]
Good idea (I like your thinking), will this be updated as new commits are added?

**Comment by archie426** [comment]
Why is this closed?

**Pull request closed without merging**